core/tests/puppeteer-acceptance-tests/spec/translation-admin-tests/add-translation-rights.spec.ts

```diff
@@ -17,24 +17,24 @@
  * rights to users.
  */
 
-const userFactory = require(
-  '../../puppeteer-testing-utilities/user-factory.js');
-const testConstants = require(
-  '../../puppeteer-testing-utilities/test-constants.js');
+import * as userFactory from
+  '../../puppeteer-testing-utilities/user-factory';
+import { ITranslationAdmin } from '../../user-utilities/translation-admin-utils';
+import testConstants from '../../puppeteer-testing-utilities/test-constants';
 
 const DEFAULT_SPEC_TIMEOUT = testConstants.DEFAULT_SPEC_TIMEOUT;
 
 describe('Translation Admin', function() {
-  let translationAdmin = null;
+  let translationAdmin: ITranslationAdmin;
```

Comment:
Do you have any thoughts on the pros and cons of declaring this as TranslationAdmin vs ITranslationAdmin? Would be glad to hear your take on that.

Reply:
I guess primarily you don't want classes to be referenced as types since here we are mainly using it for type checking. Also since we are trying to use composition and joining different classes together, interfaces are better.